sandbox/public/index.html

```diff
@@ -12,7 +12,18 @@
     <script src="https://cdn.jsdelivr.net/npm/promise-polyfill@8/dist/polyfill.min.js"></script>
 
     <title>Mock website hosting Alloy</title>
+    <script src="perf/js/visitorapi.min.js"></script>
+    <script src="perf/js/AppMeasurement.js"></script>
 
+    <script>
+      var visitor = Visitor.getInstance("97D1F3F459CE0AD80A495CBE@AdobeOrg", {
+        disableIdSyncs: true
+      });
+      var s_account = "ujslecommerce";
+      var s = s_gi(s_account);
+      // Make sure data is sent to the correct location
+      s.trackingServer = "ujsl.sc.omtrdc.net";
+    </script>
```

Comment:
Why did you need Visitor on the page? For AppMeasurement to use Visitor, you need to do `window.s.visitor = visitor`. This is working now because you do `window.s.marketingCloudVisitorID = identity.identity.ECID;` which sets up the ECID for AppMeasurement.

To be clear there are two ways for AppMeasurement to get the same visitor id as Alloy:
1. Set `s.marketingCloudVisitorID` with the result from Alloy's getIdentity call. If you do it this way there is no need to put Visitor on the page.
2. Set `s.visitor = Visitor.getInstance(...)`, and update Alloy config to have idMigrationEnabled=true.

Of course, if we can't use appMeasurement anyways because we need `tnta` to work, we'll have to generate a fetch call and we won't need appMeasurement at all.

Reply:
Good to know! thank you, I'll remove all the dependencies on AppMeadurement.js.